jobsapp/api/permissions.py

```diff
@@ -23,3 +23,14 @@ def has_object_permission(self, request: Request, view: View, obj: Any) -> bool:
         if request.method in JOB_AUTHOR_METHODS and (request.user.id == obj.user_id):
             return True
         return False
+
+
+class IsSelfOrReadOnly(BasePermission):
+    # TODO: Improve message
+    message = "Unauthorized"
```

Comment:
And add gettext

Reply:
gettext? I dont understand this

Reply:
Hi. Mark as a translatable text.

`message = _("Unauthorized")`

Reply:
You are totally right! Changed.